src/SwissPaymentSlip/SwissPaymentSlipFpdf/PaymentSlipFpdf.php

```diff
@@ -128,17 +123,19 @@ protected function convertColor2Rgb($color) {
 		return $this->rgbColors[$color];
 	}
 
-	/**
-	 * Convert hexadecimal values into an array of RGB
-	 *
-	 * @param $hexStr
-	 * @param bool $returnAsString
-	 * @param string $separator
-	 * @return array|bool|string
-	 *
-	 * @copyright 2010 hafees at msn dot com
-	 * @link http://www.php.net/manual/en/function.hexdec.php#99478
-	 */
+    /**
+     * Convert hexadecimal color code into an associative array or string of RGB values
+     *
+     * @param string $hexStr The hexadecimal color code (3 or 6 characters long)
+     * @param bool $returnAsString Whether to return as a string or array.
+     * @param string $separator The separator for the RGB value string, defaults to ",".
+     * @return array|string|false The RGB values as an associative array or a string.
+     * False if an invalid color code was given.
+     *
+     * @copyright 2010 hafees at msn dot com
+     * @link http://www.php.net/manual/en/function.hexdec.php#99478
+     * @todo Throw an exception if an invalid hex color code was given
+     */
 	private function hex2RGB($hexStr, $returnAsString = false, $separator = ',') {
```

Comment:
Doesn't this method exist in the TCPDF varaint, too?
If so we chould move it upwards.